colossalai/autochunk/select_chunk.py

```diff
@@ -63,33 +61,26 @@ def _select_fit_memory_chunk_region(
         if len(possible_chunk_regions) == 0:
             return None
 
+        max_possible_chunk_region = (min([i["region"][0] for i in possible_chunk_regions]),
+                                     max([i["region"][1] for i in possible_chunk_regions]))
+
         # get mem for chunk region
         regions_dict = []
```

Comment:
it is a dict list so the name should XXX_list?

Reply:
it is a list[dict], I will update its name